sprokit/src/sprokit/pipeline/CMakeLists.txt

```diff
@@ -56,6 +56,23 @@ set(pipeline_headers
 
 set(pipeline_private_headers)
 
+if(NOT SKBUILD)
+kwiver_make_module_path( ${CMAKE_INSTALL_PREFIX} ${kwiver_plugin_process_subdir} )
+set( sprokit_default_module_paths ${kwiver_module_path_result} )
+
+if (KWIVER_USE_BUILD_TREE)
+  kwiver_make_module_path( ${sprokit_output_dir} ${kwiver_plugin_process_subdir} )
```

Comment:
Some of this plugin path handling may be different with the new plugin directory structure where only the base directory needs to be in the path.

Reply:
I'm not sure if this changes anything, but this is currently rebased on top of those changes.

Reply:
If there is anything else to do WRT the module paths, we can address that in the future.